Resources/Prototypes/Entities/Objects/Specific/Cargo/cargo_pallet.yml

```diff
@@ -53,3 +54,117 @@
   - type: GuideHelp
     guides:
     - Cargo
+
+- type: entity
+  id: CargoPalletSell
+  name: cargo selling pallet
+  description: Designates valid items to sell with a selling computer, or to CentCom when a shuttle is recalled.
+  parent: CargoPallet
+  components:
+  - type: InteractionOutline
+  - type: Anchorable
+    flags:
+    - Anchorable
+  - type: CollideOnAnchor
+  - type: Physics
+    canCollide: false
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.45,-0.45,0.45,0.45"
+        density: 15
+        mask:
+          - MachineMask
+  - type: CargoPallet
+    pallettype: "sell"
+  - type: StaticPrice
+    price: 100
+  - type: Sprite
+    drawdepth: FloorTiles
+    layers:
+    - sprite: Structures/cargo_pallets.rsi
+      state: cargo_pallet_sell
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
+  - type: Destructible
+    thresholds:
+      - trigger:
+          !type:DamageTrigger
+          damage: 500
+        behaviors:
+          - !type:DoActsBehavior
+            acts: [ "Destruction" ]
+      - trigger:
+          !type:DamageTrigger
+          damage: 200
+        behaviors:
+          - !type:SpawnEntitiesBehavior
+            spawn:
+              PartRodMetal: # takes two to construct, so drop less than that
+                min: 0
+                max: 1
+          - !type:DoActsBehavior
+            acts: [ "Destruction" ]
+  - type: GuideHelp
+    guides:
+    - Cargo
+
+- type: entity
+  id: CargoPalletBuy
+  name: cargo buying pallet
+  description: Designates where orders will appear when purchased.
+  parent: CargoPallet
+  components:
+  - type: InteractionOutline
+  - type: Anchorable
+    flags:
+    - Anchorable
+  - type: CollideOnAnchor
+  - type: Physics
+    canCollide: false
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.45,-0.45,0.45,0.45"
+        density: 15
+        mask:
+          - MachineMask
+  - type: CargoPallet
+    pallettype: "buy"
+  - type: StaticPrice
+    price: 100
+  - type: Sprite
+    drawdepth: FloorTiles
+    layers:
+    - sprite: Structures/cargo_pallets.rsi
+      state: cargo_pallet_buy
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
+  - type: Destructible
+    thresholds:
+      - trigger:
+          !type:DamageTrigger
+          damage: 500
+        behaviors:
+          - !type:DoActsBehavior
+            acts: [ "Destruction" ]
+      - trigger:
+          !type:DamageTrigger
+          damage: 200
+        behaviors:
+          - !type:SpawnEntitiesBehavior
+            spawn:
+              PartRodMetal: # takes two to construct, so drop less than that
+                min: 0
+                max: 1
+          - !type:DoActsBehavior
+            acts: [ "Destruction" ]
+  - type: GuideHelp
+    guides:
+    - Cargo
```

Comment:
Just set the base cargo pallet as parent and override the palletType

Reply:
`- type: entity
  id: CargoPalletSell
  name: cargo selling pallet
  description: Designates valid items to sell with a selling computer, or to CentCom when a shuttle is recalled.
  parent: CargoPallet
  - type: CargoPallet
    palletType: "sell"
  - type: Sprite
    drawdepth: FloorTiles
    layers:
    - sprite: Structures/cargo_pallets.rsi
      state: cargo_pallet_sell
      
      Like this?`

Reply:
Almost! You still need the "components:" line. Sorry I didn't see you already had the parent specified in there.

When you use inheritance in entity definitions, the child will automatically get all of the components that the parent has, with the same values for all the properties. You should only list a component type again in the child if you're changing some of its values. That way, any future changes to the parent will automatically be applied to the child too

Reply:
That makes sense. Updated that in the yml, should be good to go here.